api/v1alpha1/managedocs_types.go

```diff
@@ -38,9 +38,24 @@ type ManagedOCSSpec struct {
 	ReconcileStrategy ReconcileStrategy `json:"reconcileStrategy,omitempty"`
 }
 
+type ComponentStatus string
+
+const (
+	ComponentReady   ComponentStatus = "Ready"
+	ComponentPending ComponentStatus = "Pending"
+	ComponentNone    ComponentStatus = "None"
+)
+
+type ManagedOCSComponent struct {
+	Name   string          `json:"name"`
+	Status ComponentStatus `json:"status"`
```

Comment:
Aren't we missing the timestamp field?

Reply:
Why do we need timestamp?

Reply:
Because the status of the component will always be a stale status. As such we need to know the time when the observation was true